### DISCUSSION TITLE
Error during install/upgrade process

### DISCUSSION THREAD

**Discussion by VolvoBirkenstock** [mocaccinoOS/mocaccino#111] [General] [answered]
Hi, Mocaccino Gurus.  
I tried to install layers/kdegames after performing an upgrade.  
The upgrade completed successfully and then I attempted "sudo luet install layers/kdegames".  
After running for a bit, a lot of applications were displayed with old and new versions claiming they were to be upgraded, which I found strange since I had previously completed the upgrade.  I was then provided with the following statement, to which I replied "y": 

 INFO   By going forward, you are also accepting the licenses of the packages that you are going to install in your system.                                    
 INFO   Do you want to continue with this operation? [y/N]:                                                                                                    
y   

It looked like a large upgrade started, and this was finally followed much later with the statement: 
 INFO   Checking for file conflicts..                                                                                                                          
 ERROR    Error: file conflict found: file conflict between 'layers/kde-apps-minimal-22.04.3+7' and 'layers/kdegames-22.04.3+1' ( file: usr/bin/knetwalk ) 

How do I fix this? 
Let me know if you want the complete log of activity.  
Thank you very much in advance.

**Comment by msdobrescu** [comment, marked as answer]
Hello, the package was rebuilt, please upgrade again.
The cause is the fact that it was built against a layer that was not containing that dependency, but another one contained it, so there is a conflict as a file must be delivered by one package only.
Sorry you have to upgrade again, community repo is more dynamic than desktop repo because there we add various apps according to the needs and requests.

**Comment by VolvoBirkenstock** [comment]
Thank you, @msdobrescu.  That cleared the problem. :-)
